vowpalwabbit/options_boost_po.cc

```diff
@@ -11,6 +11,9 @@
 #include <iterator>
 #include <utility>
 
+#include <boost/exception/exception.hpp>
```

Comment:
I'm assuming none of these files are 1.69 specific?

Reply:
No I don't believe so